.github/workflows/deploy.yml

```diff
@@ -0,0 +1,60 @@
+
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: [ "Build" ]
+    branches:
+      - master
+      - DEV-100_deployment-on-aws
+    types:
+      - completed
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy to Amazon ECS
+
+    steps:
+    - name: Download build artifacts
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: ${{ github.event.workflow_run.workflow_id }}
+
+    - name: Set env
+      run: |
+        echo ::set-env name=AWS_REGION::eu-central-1
+        echo ::set-env name=CLUSTER_NAME::dealog-dev
+        echo ::set-env name=SERVICE_NAME::backoffice
+        echo ::set-env name=CONTAINER_NAME::backoffice
+        echo ::set-env name=FULL_IMAGE_NAME::$(cat ./dockerized-backoffice/full_image_name.txt)
+        echo ::set-env name=IMAGE_TAG::$(cat ./backoffice/version.txt)
+    
+    - name: Set production env
```

Comment:
Where are all the application env vars (DB, Kafka etc.) set? Didn't see it.

Reply:
they are part of the ecs configuration (see task-definition.json)

Reply:
Is this file supposed to be in the repo? I don't see it.

Reply:
see download task definition below

Reply:
hm... this is not good. But I guess there is no easy way out.

Reply:
Can't we somehow make this work with GH secrets?

Reply:
the only thing that will change is the cluster_name. at the moment we have only one cluster (kiss-principle).